docs/UserGuide.md

```diff
@@ -319,6 +305,22 @@ Updates the amount of tuition fee paid by the student and the amount of tuition
 
 <div style="page-break-after: always;"></div>
 
+### Showing income data: `income`
+
+Shows the total amount of tuition fee that you have been paid, as well as total amount that is owed.
+
+**Format:** `income`
+
+**Examples**
+* `income` when all entries are listed gives the total tuition fee paid, as well as total amount that is owed
```

Comment:
I think this one can be better phrased as well

Reply:
When all entries are listed, typing `income` will ...

Or maybe you can ask gpt to phrase it for you

Reply:
Maybe can add 'Entering' at the start of the sentence.

Reply:
Thanks for the suggestion! Ive made the changes by adding entering.